test/libsolidity/smtCheckerTests/external_calls/external_hash_known_code_state.sol

```diff
@@ -34,4 +34,4 @@ contract C {
 	}
 }
 // ----
-// Warning 6328: (528-565): CHC: Assertion violation happens here.\nCounterexample:\nowner = 1, y = 0, z = 0, s = 0\n\n\n\nTransaction trace:\nconstructor()\nState: owner = 1, y = 0, z = 0, s = 0\ninv()
+// Warning 6328: (528-565): CHC: Assertion violation happens here.\nCounterexample:\nowner = 1, y = 0, z = 0, s = 0\n\n\n\nTransaction trace:\nconstructor()\nState: owner = 1, y = 0, z = 0, s = 0\nf()
```

Comment:
This is a little weird. Can the call to `s.f()` be trusted? Do we know that that source code will be executed, or we cannot know that?

Reply:
We cannot know that, I guess the comment is misleading.

Reply:
Does this example makes sense then? What should we do about it?

Reply:
I think the idea here was to make `s` trusted by `C`, maybe by adding `s = new State();` to `C`'s constructor. In that case the comment makes sense.